app/views/static.py

```diff
@@ -1,25 +1,43 @@
-from flask import Blueprint
+from flask import Blueprint, g, current_app
+from flask_login import current_user
 from flask_themes2 import render_theme_template
 
-from app.helpers import template_helper
-
+from app.globals import get_metadata
+from app.utilities.schema import load_schema_from_metadata
+from app.data_model.session_data import SessionData
 
 from structlog import get_logger
 
 logger = get_logger()
 
-contact_blueprint = Blueprint('contact', __name__,)
+contact_blueprint = Blueprint(name='contact', import_name=__name__)
 
 
 @contact_blueprint.route('/contact-us', methods=['GET'])
 def contact():
-    return render_theme_template('default', 'static/contact-us.html')
+    metadata = get_metadata(current_user)
+    g.schema = load_schema_from_metadata(metadata)
+
+    session_data = SessionData(
```

Comment:
Session data is already held in the session_store which you can get from globals.py (get_session_store). questionnaire.py uses this if you'd like an example. You can then get the schema using the eq-id and form_type from the session data.